appserver/admingui/common/src/main/resources/shared/sslPrepare.inc

```diff
@@ -61,8 +61,6 @@ if (! #{requestScope.exist}){
       -->
     createMap(result="#{pageSession.valueMap}");
     mapPut(map="#{pageSession.valueMap}" key="ssl3Enabled" value="false");
```

Comment:
We should increase the scope of this to remove SSL3 too - I didn't realise it was even still there.
I'll update the Jira